tests/unit/modules/test_roles.py

```diff
@@ -0,0 +1,304 @@
+import json
+from unittest.mock import Mock
+from unittest.mock import patch
+
+import pytest
+import saltext.vmware.modules.roles as security_roles
+import saltext.vmware.utils.drift as drift
+from pyVmomi import vim
+
+
+def mock_with_name(name, *args, **kwargs):
+    # Can't mock name via constructor: https://docs.python.org/3/library/unittest.mock.html#mock-names-and-the-name-attribute
+    mock = Mock(*args, **kwargs)
+    mock.name = name
+    return mock
```

Comment:
**suggestion (non-blocking)** when this change https://github.com/Onemind-Services-LLC/salt-ext-modules-vmware/pull/7/commits/8117573af2928b1c717ecdf4b9ff6f165a426d80 gets merged in this will want to change. TBH if that PR doesn't get updated by tomorrow afternoon I'll probably go ahead and split that out.